### ISSUE TITLE
Possible error in text classification example code for multi-label classification

### ISSUE THREAD

**Issue by subhalingamd** [huggingface/transformers#26830]
### System Info

N/A *(system config independent)*

### Who can help?

@ArthurZucker and @younesbelkada 

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [X] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

N/A

### Expected behavior

Shouldn't the default condition be `p > 0` in the following lines for multi-label classification? The sequence classification model returns the logits and sigmoid has not been applied anywhere.

https://github.com/huggingface/transformers/blob/5c081e29930466ecf9a478727039d980131076d9/examples/pytorch/text-classification/run_classification.py#L658

https://github.com/huggingface/transformers/blob/5c081e29930466ecf9a478727039d980131076d9/examples/pytorch/text-classification/run_classification.py#L724

**Comment by younesbelkada** [comment]
True yes, this seems to be a good point - cc @ranchlai who added this script

**Comment by ranchlai** [comment]
Indeed. This will cause higher precision and lower recall in multi-label classification. Thanks @subhalingamd for pointing out. I can give a quick fix. what do you think? @younesbelkada

**Comment by younesbelkada** [comment]
That would be great @ranchlai ! Thanks a lot !
